webapp/src/components/Metrics/PerformanceAnalysisTable.tsx

```diff
@@ -0,0 +1,642 @@
+  ) : (
+    <Paper
+      variant="outlined"
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        height: "100%",
+        padding: 4,
+        marginTop: 4,
+      }}
+    >
+      <Box marginBottom={2} display="flex" justifyContent="space-between">
+        <Box width={340}>
+          <DatasetSplitToggler
+            availableDatasetSplits={availableDatasetSplits}
+            value={selectedDatasetSplit}
+            onChange={setSelectedDatasetSplit}
+          />
+        </Box>
+        {config && (
+          <Box display="flex" flexDirection="row" alignItems="center">
+            <FormControlLabel
+              label={`Compare Baseline (${
+                config.pipelines?.[pipeline.pipelineIndex].name
+              }) with:`}
+              labelPlacement="start"
+              sx={{ gap: 1, paddingRight: 2 }}
+              control={
+                <PipelineSelect
+                  selectedPipeline={comparedPipeline}
+                  onChange={setComparedPipeline}
+                >
+                  {config.pipelines?.map(
+                    ({ name }, pipelineIndex) =>
+                      pipelineIndex !== pipeline.pipelineIndex && (
+                        <MenuItem key={pipelineIndex} value={pipelineIndex}>
+                          {name}
+                        </MenuItem>
+                      )
+                  )}
+                </PipelineSelect>
+              }
+            ></FormControlLabel>
+          </Box>
+        )}
+      </Box>
+      <Table
+        sx={{
+          "& .MuiDataGrid-cell": {
+            borderBottom: "none",
+          },
+          "& .MuiDataGrid-columnHeaders": {
+            borderBottom: "none",
+            height: DATA_GRID_PROPS_DEFAULT_VALUES.headerHeight,
+          },
+          "& .MuiDataGrid-virtualScroller": {
+            // Stops accidental navigation on horizontal scroll with touch pad
+            overscrollBehaviorX: "contain",
+          },
+          "& .delta": {
+            background: (theme) => theme.palette.grey[100],
+          },
+          "& .sticky": {
+            position: "sticky",
+            left: 0,
+            zIndex: (theme) => theme.zIndex.mobileStepper,
+            background: (theme) => theme.palette.background.paper,
+          },
+
+          "& .overall": {
+            background: (theme) => theme.palette.grey[200],
+          },
+          "& .no-border-right": {
+            borderRight: "none",
+          },
+        }}
+        showCellRightBorder
+        showColumnRightBorder
+        sortModel={sortModel}
+        onSortModelChange={setSortModel}
+        columnVisibilityModel={columnVisibilityModel}
+        onColumnVisibilityModelChange={setColumnVisibilityModel}
+        getRowClassName={({ id }) =>
+          `${id === OVERALL_ROW_ID ? "overall" : ""}`
+        }
+        getCellClassName={({ id }) =>
+          `${id === OVERALL_ROW_ID ? "overall" : ""}`
+        }
```

Comment:
The last point from my suggestion in https://github.com/ServiceNow/azimuth/pull/228#discussion_r988421070 is incomplete. The `getRowClassName` is not useful anymore. Also, it is unnecessary to wrap the ternary operator inside a `` `${...}` ``, so you can simplify.
```suggestion
        getCellClassName={({ id }) => (id === OVERALL_ROW_ID ? "overall" : "")}
```

Reply:
Don't forget to also remove the unnecessary `` `${...}` ``.